recipes/deepinv/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "deepinv" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deepinv-{{ version }}.tar.gz
+  sha256: 1b44b473544e6f2998e7f80dbcb94136a3528e06dfa53b5c00c2d4c2d0bc4ec1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=61.2
+    - pip
+  run:
+    - python >=3.8
+    - numpy
+    - matplotlib-base
+    - hdf5storage
+    - tqdm
+    - pytorch
+    - torchvision
+    - einops
+    - wandb
+
+test:
+  imports:
+    - deepinv
+  commands:
+    - pip check
+    
+  requires:
+    - pip
+    - pytest
+    - pytest deepinv/tests -vs
```

Comment:
Move this line over to `commands`.  This section is only for the test dependencies.

```suggestion
```